iridium-core-server/src/main/java/software/iridium/api/ApiExceptionHandler.java

```diff
@@ -30,13 +31,20 @@ public class ApiExceptionHandler {
 
   @ResponseStatus(value = HttpStatus.UNAUTHORIZED)
   @ExceptionHandler(NotAuthorizedException.class)
-  public @ResponseBody ApiResponse handleNotAuthorizedException(final Exception e) {
-    return new ApiResponse(HttpStatus.UNAUTHORIZED.toString(), e.getMessage());
+  public @ResponseBody ModelAndView handleNotAuthorizedException(final NotAuthorizedException e) {
+    ModelAndView modelAndView = new ModelAndView();
```

Comment:
There are some cases where we still need to return the `NotAuthorizedException` in JSON format.  Will this change make every `NotAuthorizedException` thrown to return the login page?

Reply:
Yes, this implementation would always return the error page. We could check the `Accept` header to decide whether JSON or the error page should be returned. WDYT about that? Do you've an approach on your mind?